proposals/witx/docs.md

```diff
@@ -0,0 +1,857 @@
+# Types
+## <a href="#size" name="size"></a> `size`: `usize`
+
+## <a href="#errno" name="errno"></a> `errno`: Enum(`u16`)
+Error codes returned by functions.
+Not all of these error codes are returned by the functions provided by this
+API; some are used in higher-level library layers, and others are provided
+merely for alignment with POSIX.
+
+### Variants
+- <a href="#errno.success" name="errno.success"></a> `success`
+No error occurred. System call completed successfully.
+
+- <a href="#errno.2big" name="errno.2big"></a> `2big`
```

Comment:
originally all errorno constants has `E` prefix, so `E2BIG` is ok. But in most languages leading digit for identifier not allowed so perhaps better use `toobig` or `_2big` in this case?

Reply:
Hi, this is from [the original `typenames.witx` file](https://github.com/WebAssembly/WASI/blob/master/phases/ephemeral/witx/typenames.witx#L42) from the WASI repository; maybe you might want to report it there.

Reply:
Hi, yes, I also dislike that the identifier is `2big`, we should have made it `toobig` upstream. I will work on getting this fixed.